app/views/journal_articles/edit_fields/_default.html.temp.erb

```diff
@@ -0,0 +1,6 @@
+<% # the hyrax version of this file messes singular field display %>
```

Comment:
can you state that this is an override file, and list which version of hyrax you're overriding?

Reply:
ref: https://github.com/scientist-softserv/adventist-dl/pull/398#issuecomment-1542281641